app/models/interactive_page.rb

```diff
@@ -143,10 +147,10 @@ def add_embeddable(embeddable, position = nil, section = nil)
     end
   end
 
-  def add_interactive(interactive, position = nil, validate = true)
+  def add_interactive(interactive, position = nil, validate = true, section = INTERACTIVE_BOX)
```

Comment:
Is this change necessary?

Reply:
Yes, this allows us pass `nil` to add the interactive to the assessment block here: https://github.com/concord-consortium/lara/pull/596/files#diff-85f32c83abf3ed472f1a05afc00bcd5eR83

Reply:
It is probably better to call add_embeddable directly in that case.

This method is only used in tests. And it shows the interactive section when it is called.
Ideally we'd remove this method and move it to a test helper.